extra/grafana/README.md

```diff
@@ -106,27 +123,30 @@ Let's take a walkthrough an example of scraping the `/metrics/array` endpoint.
         site: London
         env: production
 
-# Repeat for the above for additional endpoints but note the limitations of collecting these in https://github.com/PureStorage-OpenConnect/pure-fb-openmetrics-exporter.
+# Repeat for the above for additional endpoints (if required) but note the limitations of collecting these in https://github.com/PureStorage-OpenConnect/pure-fb-openmetrics-exporter.
 # /metrics/usage
 # /metrics/clients 
 
 # Repeat again for more Pure Storage FlashBlades
 ```
 4. Test the prometheus.yaml file is valid
-```console
-> promtool check config /etc/prometheus/prometheus.yml
-Checking prometheus.yml
- SUCCESS: prometheus.yml is valid prometheus config file syntax
-```
+
+    ```console
+    > promtool check config /etc/prometheus/prometheus.yml
+    Checking prometheus.yml
+    SUCCESS: prometheus.yml is valid prometheus config file syntax
+    ```
+
 5. Restart Prometheus to ensure changes take effect
 
 6. Navigate to your Prometheus instance via web browser http://prometheus:9090
```

Comment:
you reference this as `prometheus-server.fqdn.com` in the patch. You should try to be consistent

Reply:
Corrected in https://github.com/PureStorage-OpenConnect/pure-fb-openmetrics-exporter/pull/44/commits/c03ac9cef7e19f3d91287260f58199b6bcb74233